src/parse/values/multiple-choice/parser.pegjs

```diff
@@ -0,0 +1,67 @@
+start
+  = text:text lineWs* newline choices:choices
+  {
+    return {
+      __type__: 'multiple-choice',
+      text: text,
+      choices: choices
+    };
+  }
+
+
+choices 'choices'
+  = first:choice rest:(newline c:choice lineWs* { return c; })*
+  { return [first].concat(rest); }
+
+
+choice 'choice'
+  = digit+ '. ' text:choiceText lineWs* name:choiceName? lineWs*
+  {
+    return {
+      text: text,
+      name: name || null
+    };
+  }
+
+choiceName 'choice name'
+  = '{=' name:symbol '}'
+  { return name; }
+
+
+symbol 'symbol'
+  = lcletter (lcletter / digit / dash)+
+  { return text(); }
```

Comment:
Since this has to match the symbol definition in the main parser, I wonder if there is a way to share the definition between the two?